src/cpp/benders/benders_core/include/core/ILogger.h

```diff
@@ -70,7 +71,8 @@ struct LogData {
            relative_gap == lhs.relative_gap &&
            max_iterations == lhs.max_iterations &&
            benders_elapsed_time == lhs.benders_elapsed_time &&
-           master_time == lhs.master_time;
+           master_time == lhs.master_time &&
+           subproblem_time == lhs.subproblem_time;
```

Comment:
The compiler can generate this operator for us with the exact same behavior : compare all data members. I counted 16 data members and 16 `==` comparisons. Unless you want to ignore some data members in the comparison, I suggest you remove lines 64-75.

To add some weight to my argument, `master_time` was previously forgotten, you added it in this commit.

Reply:
You're right. C++20 introduced default `operator==` though. Why do you need this comparison operator ? Tests ?

Reply:
yep only for tests.